local/alpha-sdk.sh

```diff
@@ -0,0 +1,155 @@
+    if [[ $# -gt 0 ]]; then
+        >&2 echo "Error: $*"
+    fi
+    exit 1
+}
+
+usage() {
+    cat <<EOF
+
+Usage:
+
+    --bottlerocket-dir            The directory of the Bottlerocket checkout that we will build.
+    --variant                     The Bottlerocket variant that we will build.
+    --sdk-version                 The version of the Bottlerocket SDK to use when building packages
+                                  and to use as the base for the Twoliter alpha-sdk. For example if
+                                  the SDK version is v0.50.0 then --sdk-version should be the same
+                                  (i.e. with the v prefix).
+    --sdk-name                    The name prefix of the SDK. For example, in this following string
+                                  'bottlerocket' is the SDK name:
+                                  public.ecr.aws/bottlerocket/bottlerocket-sdk-x86_64:v0.50.0
+                                  Note that the suffix '-sdk' is assumed and added to the name,
+                                  which is just 'bottlerocket'
+    --sdk-registry                The namespace or Docker registry where the SDK is found. For
+                                  example, in the following string 'public.ecr.aws' is the
+                                  registry:
+    --alpha-name                  The name of the Twoliter alpha SDK container.
+                                  public.ecr.aws/bottlerocket/bottlerocket-sdk-x86_64:v0.50.0
+    --alpha-registry              The registry to which the resultant alpha SDK images will be
+                                  pushed.
+    --alpha-version               The version the alpha SDK should be tagged with. In practice this
+                                  should match the Bottlerocket version of the packages being built.
+    -h, --help                    Show this help text
+
+EOF
+}
+
+usage_error() {
+    >&2 usage
+    bail "$1"
+}
+
+#
+# Parse arguments
+#
+
+while [[ $# -gt 0 ]]; do
+    case $1 in
+        --bottlerocket-dir)
+            shift; bottlerocket_dir=$1 ;;
+        --variant)
+            shift; variant=$1 ;;
+        --sdk-version)
+            shift; sdk_version=$1 ;;
+        --sdk-name)
+            shift; sdk_name=$1 ;;
+        --sdk-registry)
+            shift; sdk_registry=$1 ;;
+        --alpha-registry)
+            shift; alpha_registry=$1 ;;
+        --alpha-name)
+            shift; alpha_name=$1 ;;
+        --alpha-version)
+            shift; alpha_version=$1 ;;
+        -h|--help)
+            usage; exit 0 ;;
+        *)
+            usage_error "Invalid option '$1'" ;;
+    esac
+    shift
+done
+
+set -e
+
+[[ -n ${bottlerocket_dir} ]] || usage_error 'required: --bottlerocket-dir'
+[[ -n ${sdk_version} ]] || usage_error 'required: --sdk-version'
+[[ -n ${alpha_registry} ]] || usage_error 'required: --alpha-registry'
+[[ -n ${alpha_version} ]] || usage_error 'required: --alpha-version'
+
+variant="${variant:=aws-k8s-1.27}"
```

Comment:
Let's default to one with all the new feature flags enabled:
```suggestion
variant="${variant:=aws-dev}"
```